stable/kiam/templates/agent-daemonset.yaml

```diff
@@ -8,6 +8,13 @@ metadata:
     component: "{{ .Values.agent.name }}"
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
+    {{- if .Values.agent.daemonsetLabels }}
+      {{- toYaml .Values.agent.daemonsetLabels | nindent 4 }}
```

Comment:
For consistency could you please move to the left hand margin and increase the indent count:
```
    {{- if .Values.agent.daemonsetLabels }}
{{- toYaml .Values.agent.daemonsetLabels | nindent 8 }}
    {{- end }}
```

Reply:
why do i need to increase the indent count?  4 spaces is the amount of indent i want ttbomk